corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -17,15 +17,16 @@
            clickable: collapsible
         },
         attr: collapsible ? {
-                role: 'button',
+                'role': 'button',
                 'aria-expanded': showChildren() ? 'true' : 'false',
-                'aria-labelledby': captionId()
+                'aria-labelledby': captionId(),
+                'tabindex': ko.utils.unwrapObservable($data.caption_markdown) || caption() ? '0' : '-1'
```

Comment:
Would be nice to add a comment on why we are doing this in the code since there is no way one would figure out the reason behind this without considerable effort.